jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -186,6 +186,10 @@ private Unsafe() {}
 	/* Mask byte offset of an int. */
 	private static final long BYTE_OFFSET_MASK = 0b11L;
 
+	/*[IF INLINE-TYPES]*/
+	private static final Object inlineTypesLock = new Object();
```

Comment:
do `= new Object() {};` instead. j.l.Object will become an interface in the future

Reply:
addressed in 929154f

Reply:
Wouldn't it be helpful to give that class a name, like is done elsewhere, for example, `ClassLoader.LazyInitLock`?

Reply:
https://github.com/eclipse-openj9/openj9/issues/14754